pkg/controller/scyllacluster/sync_certs.go

```diff
@@ -191,23 +191,80 @@ func (scc *Controller) syncCerts(
 
 	// For the clients not using the proxy we'll sign for IPs that are published by ScyllaDB.
 	var ipAddresses []net.IP
-	for _, svc := range serviceMap {
-		if svc.Labels[naming.ScyllaServiceTypeLabel] != string(naming.ScyllaServiceTypeMember) {
-			continue
-		}
+	var servingDNSNames []string
 
-		ips := svc.Spec.ClusterIPs
-		if len(ips) == 0 {
-			klog.V(2).InfoS("No IP available for member", "Service", klog.KObj(svc))
-			continue
-		}
-		parsedIPs, err := helpers.ParseClusterIPs(ips)
-		if err != nil {
-			return progressingConditions, err
+	if sc.Spec.ExposeOptions != nil && sc.Spec.ExposeOptions.BroadcastOptions != nil && sc.Spec.ExposeOptions.BroadcastOptions.Clients != nil {
+		switch sc.Spec.ExposeOptions.BroadcastOptions.Clients.Type {
```

Comment:
should we sign them for all that are available? say a clusteIP should work as an initial contact point even if you publish PodIPs and vice versa

Reply:
done